CMakeLists.txt

```diff
@@ -4,9 +4,12 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake-modules")
 
 set(CMAKE_C_STANDARD 11)
 
-option(BUILD_TESTS "Build and run tests" OFF)
-option(TESTS_COV "Run coverage on tests" OFF)
-option(BUILD_SINGLE_HEADER "Build single-header version" OFF)
+option(BUILD_TESTS "Build and run tests"                        OFF)
+option(TESTS_COV "Run coverage on tests"                        OFF)
+option(BUILD_SINGLE_HEADER "Build single-header version"        OFF)
+option(BUILD_WINDOWS_EXAMPLE "Builds Win32 example application" OFF)
```

Comment:
I think it's better to have single flag for all example applications (even though only one is currently configured from cmake). Let's change this flag to `BUILD_EXAMPLES`. Also you should enable this flag in github CI when configuring CMake project:
https://github.com/funbiscuit/embedded-cli/blob/master/.github/workflows/ci.yaml#L40
https://github.com/funbiscuit/embedded-cli/blob/master/.github/workflows/ci.yaml#L108
https://github.com/funbiscuit/embedded-cli/blob/master/.github/workflows/ci.yaml#L144

Even though there is no examples for other platforms.